.github/workflows/snyk_scan.yaml

```diff
@@ -0,0 +1,30 @@
+---
+name: vanagon check PRs
+
+# on:
+#   pull_request_target:
+#     types: ['opened','edited','reopened','synchronize']
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+ snyk_vanagon:
+   runs-on: ubuntu-latest
+   steps:
+    - name: checkout the current PR
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        persist-credentials: false
+    - name: Run Vanagon Snyk Scan
+      uses: puppetlabs/security-snyk-vanagon-action@v1.0.0
+      with:
+        snykToken: ${{ secrets.SNYK_PE_TOKEN }}
+        snykOrg: 'puppet-enterprise'
+        skipProjects: 'agent-runtime-5.5.x,agent-runtime-1.10.x,client-tools-runtime-irving,pdk-runtime'
+        skipPlatforms: 'cisco-wrlinux-5-x86_64,cisco-wrlinux-7-x86_64,debian-10-armhf,eos-4-i386,fedora-30-x86_64,fedora-31-x86_64,osx-10.14-x86_64'
```

Comment:
I'm confused, does this means that we skip `agent-runtime-5.5.x` on all platforms or we skip it only on `skipPlatforms`?

I think it will be more natural to specify for each project the list of platforms to build/inspect on. If this is planned on future iteration, you can ignore my comment for now.

Reply:
we can definitely change to that model. RIght now the action:

* gets all projects except for those in skipProjects
* gets all platforms except for those in skipPlatforms
* runs the test for all permutations of `projects` x `platforms`

This was easier to get out this door this way, but I'm open to changing the model